README.md

```diff
@@ -27,16 +28,18 @@ Depending on the type of models configured, the module implements:
 ## Configure your `torchvision` vision service
 
 > [!NOTE]  
-> Before configuring your vision service, you must [create a robot](https://docs.viam.com/manage/fleet/robots/#add-a-new-robot).
+> Before configuring your vision service, you must [create a machine](https://docs.viam.com/how-tos/configure/).
+
+Navigate to the [**CONFIGURE** tab](https://docs.viam.com/configure/) of your [machine](https://docs.viam.com/fleet/machines/) in the [Viam app](https://app.viam.com/).
+[Add vision / torchvision to your machine](https://docs.viam.com/configure/#components).
 
-Navigate to the **Config** tab of your robot’s page in [the Viam app](https://app.viam.com/). Click on the **Services** subtab and click **Create service**. Select the `Vision` type, then select the `torchvision` model. Enter a name for your service and click **Create**.
+### Example configuration with a camera and transform camera
 
-### Example of config with a camera and transform camera
-The following json config file includes the following ressources:
-- TorchvVision module
-- modular ressource (Torchvision vision service)
--  a camera
-- a transform camera 
+The following JSON config file includes the following resources:
+- TorchVision module
+- modular resource (TorchVision vision service)
+- a webcam [camera](https://docs.viam.com/components/camera/)
```

Comment:
What is the objective of including this link? Should it be this?
```suggestion
- a webcam [camera](https://docs.viam.com/components/camera/webcam/)
```

Reply:
originally it was just camera here (although configuration is webcam) so I thought having the link to just camera might be helpful bc of the camera api. this is also fine though and more standardized!

Reply:
okay whatever you think makes the most sense--you have more context than I do